libexec/scoop-update.ps1

```
@@ -294,9 +294,19 @@ function update($app, $global, $quiet = $false, $independent, $suggested, $use_c
     Write-Host "Updating '$app' ($old_version -> $version)"
 
     #region Workaround for #2952
-    if (test_running_process $app $global) {
+    $running_ret = test_running_process $app $global
+    $stopped_services = @()
+    $stopped_processes = @()
+    if ($running_ret -eq $true) {
         Write-Host 'Running process detected, skip updating.'
```

PSScriptAnalyzer (1.24.0)

[warning] File 'scoop-update.ps1' uses Write-Host. Avoid using Write-Host because it might not work in all hosts, does not work when there is no host, and (prior to PS 5.0) cannot be suppressed, captured, or redirected. Instead, use Write-Output, Write-Verbose, or Write-Information.

(PSAvoidUsingWriteHost)